src/lib/prepareOas.ts

```diff
@@ -1,9 +1,8 @@
 import fs from 'fs';
 
 import chalk from 'chalk';
-import ora from 'ora';
-
 import OASNormalize from 'oas-normalize';
```

Comment:
This wasn't being sorted before `ora` before because the sinkhole exclusion was causing the `import/order` rule to think that it was a local package.